src/main/java/com/sprint/mission/discodeit/service/basic/BasicChannelService.java

```diff
@@ -29,7 +30,6 @@ public class BasicChannelService implements ChannelService {
 
     private final ChannelMapper channelMapper;
 
-
     @Transactional
     @Override
     public ChannelDto create(PrivateChannelCreateRequest request) {
```

Comment:
@coderimspace 데이터 변경시에 캐시 무효화 정책이 필요할 거 같은데 아닐까요?
```suggestion
    @CacheEvict(cacheNames = "channelList", key = "#userId")
    public ChannelDto create(PrivateChannelCreateRequest request) {
```